common/changes/office-ui-fabric-react/nasabek-fixAligmentProblem_2019-06-25-23-13.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "create new function in the focuszone which control the alignment state to decide if we want to keep the previous alignment or we need to update it depends on the position of the new active element",
```

Comment:
Could we have this change file follow this template? `ComponentName: what behavior was fixed or changed.`

So, here we could write something like the following:

`FocusZone: add setForceAlignmentState method to determine whether to keep the previous alignment or update it based on the position of the new active element`